scripts/extensions/sams/src/components/assets/assetEditor.tsx

```diff
@@ -141,6 +169,42 @@ class AssetEditorComponent extends React.PureComponent<IProps> {
                         </FormRow>
                     </FormGroup>
                 )}
+                {this.fieldEnabled('tags') && (
+                    <FormGroup>
+                        <FormRow>
+                            <Autocomplete
+                                key={nextId.name}
```

Comment:
It doesn't look like this `key` is being changed. After clicking on a suggested item from the popup, the text field in Autocomplete should clear.